src/client/LoginComponent.jsx

```diff
@@ -26,7 +26,7 @@ const LoginComponent = ({isLoggedIn, userHydrated, urls, mode, nodePath, isShowR
     };
 
     const logout = () => {
-        fetch('/cms/logout');
+        fetch(urls.logoutUrl);
```

Comment:
This promise is ["floating"](https://typescript-eslint.io/rules/no-floating-promises/), it might be a mistake

It would be best to rewrite it as such:

```js
const logout = async () => {
  await fetch(...);
  setLoggedIn(false);  
}
```

to handle errors and prevent "race conditions" (not really one but huh).

I would tend to favor an imperative style over the functional: `fetch(...).then(() => setLoggedIn(false))`, but both would do the trick